slack/README.md

```diff
@@ -0,0 +1,121 @@
+# slack
+
+This library allows using Slack with Wing.
+
+## Prerequisites
+
+* [winglang](https://winglang.io)
+
+## Installation
+
+Use `npm` to install this library:
+
+```sh
+npm i @winglibs/slack
+```
+## Bring it
+
+```js
+bring slack;
+bring cloud;
+
+let myBotToken = new cloud.Secret(name: "mybot_token");
+
+let slackbot = new slack.App(botToken: myBotToken);
```

Comment:
Sorry for the hassle. I am now realizing this is called a bot in the slack api, so we might as well call this `slack.Bot`

Sorry!

Reply:
Are you sure? Im pretty sure you were right the first time this is an `App` which can be used to describe a bot

Reply:
I am not sure :-)

So let's leave it `App`. Sorry!